src/app/files/new-files/new-files.component.ts

```diff
@@ -206,16 +205,10 @@ export class NewFilesComponent implements OnInit, OnDestroy {
    * @returns {{ update: { filename: string, isSecret: boolean, fileType: number, accessGroupId: number, sourceType: string, sourceData: string }, status: boolean }} Prepared form data.
    */
   onBeforeSubmit(form: any, status: boolean) {
-    let sourcadata;
-    let fname;
-
-    if (form.sourceType === 'inline') {
-      fname = form.filename;
-      sourcadata = Buffer.from(form.sourceData).toString('base64');
-    } else {
-      sourcadata = this.fileName;
-      fname = this.fileName;
-    }
+    const sourceType = form.sourceType || 'import';
+    const isInline = sourceType === 'inline';
+    const fname = isInline ? form.filename : this.fileName;
```

Comment:
Why is filename camelCased sometimes? :)

Reply:
Thanks. Changes to camelCased